Lesson9/organizer9/scheduler/forms.py

```diff
@@ -0,0 +1,60 @@
+# scheduler/forms.py
+# forms.py should contain fields input check
+from django import forms
+from django.core.exceptions import ValidationError
+from .models import Note
+
+
+class NoteForm(forms.Form):
+    title = forms.CharField(max_length=20)
+    msg = forms.CharField(max_length=100)
+    assignee = forms.CharField(max_length=100, required=False)
+    e_mail = forms.EmailField(required=False)
+
+    def clean_title(self):
+        cln_title = self.cleaned_data['title']
+
+        if len(cln_title.split(' ')) < 2:
+            raise ValidationError("Use two words Title")
+        return cln_title
+
+    def clean_assignee(self):
+        # Whole assignee field:
+        cln_assingnee = self.cleaned_data['assignee']
+        # List of words in assignee field:
+        words = cln_assingnee.split()
+
+        # This check makes it possible for field to be Optional:
+        if cln_assingnee:
+            # Amount of Letters in Assignee /or/ Amount of words:
+            if len(cln_assingnee) < 5 or len(words) < 2:
```

Comment:
min_length is better to check on the field validation constraints than in the clean method.

Both conditions could be checked in one line since there are no any additional logical branches.
```suggestion
        if words and len(words) < 2:
```